src/Umbraco.Web.UI/Umbraco/config/lang/en.xml

```diff
@@ -2531,9 +2531,10 @@ To manage your website, simply open the Umbraco back office and start adding con
     <key alias="endTitle">End preview mode</key>
     <key alias="openWebsiteLabel">Preview website</key>
     <key alias="openWebsiteTitle">Open website in preview mode</key>
-    <key alias="returnToPreviewHeadline">Preview content?</key>
-    <key alias="returnToPreviewDescription">You have ended preview mode, do you want to continue previewing this content?</key>
-    <key alias="returnToPreviewButton">Preview</key>
+    <key alias="returnToPreviewHeadline">Preview website?</key>
+    <key alias="returnToPreviewDescription">ou have ended preview mode, do you want to enable it again to view the latest saved version of your website?</key>

```

Comment:
Small typo here (and on en_us.xml)

Reply:
Glad someone got their glasses on. Thanks for the spot @rbottema

Reply:
@rbottema thanks a lot, good spotting, I have made a fix for it :-)